docs/tutorial.md

```diff
@@ -0,0 +1,396 @@
+# The Simplest Thing That Can Possibly Work
+
+This repository template contains a lot of material, and provides a
```

Comment:
Do we think we need this paragraph at all?

Reply:
Removed